components/ActionSheet/tests/unit/MDCActionSheetControllerDelegateTests.m

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+#import "MDCActionSheetTestHelper.h"
+
+@interface ActionSheetControllerDelegate : NSObject <MDCActionSheetControllerDelegate>
+@property(nonatomic, readonly) MDCActionSheetController *dismissedActionSheetController;
+@end
+
+@implementation ActionSheetControllerDelegate
+- (void)actionSheetControllerDidDismiss:(MDCActionSheetController *)actionSheetController {
+  _dismissedActionSheetController = actionSheetController;
+}
+@end
+
+@interface MDCActionSheetControllerDelegateTests : XCTestCase
+/** The @c MDCActionSheetController being tested. */
+@property(nonatomic, strong) MDCActionSheetController *actionSheet;
+
+/** An @c MDCActionSheetControllerDelegate to verify delegate functions are called. */
+@property(nonatomic, nullable) ActionSheetControllerDelegate *actionSheetControllerDelegate;
+@end
+
+@implementation MDCActionSheetControllerDelegateTests
+
+- (void)setUp {
```

Comment:
Since there is just one test here should we remove the `setUp` and `tearDown` methods and just put the `setUp` in the `given` section of the test?

Reply:
I thought about that, but because this file is specifically to test the delegate, this seems like it would be necessary for any test we'd want to put here.